Framework/Algorithms/src/CreateWorkspace.cpp

```diff
@@ -47,7 +46,7 @@ void CreateWorkspace::init() {
   declareProperty(Kernel::make_unique<ArrayProperty<double>>("DataY", required),
                   "Y-axis data values for workspace (measures).");
   declareProperty(make_unique<ArrayProperty<double>>("DataE"),
-                  "Error values for workspace. Optional.");
+                  "Error values for workspace.");
```

Comment:
Is it not optional? (removed accidentally?)

Reply:
I think this will not be visible anyway and I thought that it is clear that this is a not required input option. For the Dx values, they are as well not required input values but will now exist in the output workspace. That's why I thought leaving the optional for Dx and not for E. But if this is not clear I do not mind at all to put it back to the E values tooltip?

Reply:
You are right, there are other indicators in the GUI.

Reply:
Thanks for your considerations.